### ISSUE TITLE
error: reading email list

### ISSUE THREAD

**Issue by Virgula0** [tatanus/SPF#16]
Hi,

I'm trying to use your script to send the web phishes pages by emails and I'm trying with some my email.... So, I create a txt file and call it email.txt that contains emails for three different lines, in the Spf directory...

When I try to launch the attack I type:

./spf.py -d example.com --ip my_public_address -f email.txt -C default.cfg

When the attack start appears me emails that are included in file: "email.txt"..
Then for the question : "obtaining list of email address" I answer "Y", the program write the emails included in the file and it automatically exit and don't laugh the attack...

Why ?? there is an error or am I that do something wrong? 

Thanks!

**Comment by tatanus** [comment]
you need to add the "-s" flag to send the emails.

from the help page: [python spf.py -h]
-s                   enable automated sending of phishing emails to targets

**Comment by Virgula0** [comment]
Ok i will try soon

**Comment by Virgula0** [comment]
I tried adding -s flag but it Doesn't work
![Image](http://tinypic.com/r/120rn7r/9)

**Comment by tatanus** [comment]
sorry for the delay and the issue.  In the current implementation, the phishing emails have to point to a phishing website.  to enable the generation of the phishing websites, use the "-w" option.  You can also use the "-v -v" options to enable verbose/debug output.

This project has been slightly neglected by me over the past year while I was working on a different project "APT2".  I do plan on revisiting SPF to update and improve it in the next few months.

**Comment by Virgula0** [comment]
ok, so I'm waiting for your updates

**Comment by tatanus** [comment]
did you try running it with the "-w" option?

As for the updates, be patient, they may take a few months.  It is starting in on the holiday season after all. :-)

**Comment by Virgula0** [comment]
yes I tried but it returns me that couldn't send email at targets

**Comment by tatanus** [comment]
is this still an issue?  just checking as I was unable to reproduce your error and several code update/changes have been made since the last comment.  Just looking to see if this is still an issue and if so, I will take another look to see if I can figure it out.
